graphql_api/views.py

```diff
@@ -50,7 +93,8 @@ async def post(self, request, *args, **kwargs):
         log.info("GraphQL Request", extra=log_data)
 
         # request.user = await get_user(request) or AnonymousUser()
-        return await super().post(request, *args, **kwargs)
+        with RequestFinalizer():
+            return await super().post(request, *args, **kwargs)
```

Comment:
What's a way for us to see/measure a connection being closed through this code?

Reply:
What do you mean? All requests comes to this code.

Reply:
Like how do we know this code indeed closed a connection after the fact, is there a utility or metric that shows live connection before, during and after?